src/Sulu/Bundle/AdminBundle/Behat/AdminContext.php

```diff
@@ -0,0 +1,341 @@
+        items[i].parentNode.click();
+    }
+};
+EOT
+        ;
+        $script = sprintf($script, $arg2, $arg1);
+        $this->getSession()->executeScript($script);
+    }
+
+    /**
+     * Fill in a husky text field
+     * @Given I fill in husky field :name with :value
+     */
+    public function iFillTheHuskyField($name, $value)
+    {
+        $this->fillInHuskyField($name, $value);
+    }
+
+    /**
+     * @Given I fill in husky field :name with :value in the overlay
+     */
+    public function iFillTheHuskyFieldInTheOverlay($name, $value)
+    {
+        $this->fillInHuskyField($name, $value, '.husky-overlay-container ');
+    }
+
+    /**
+     * @Then I click the ":text" button
+     */
+    public function iClickTheButton($text)
+    {
+        $script = $this->clickByTitle('.btn', $text);
+    }
+
+    /**
+     * @Then I click the column navigation item :itemTitle
+     */
+    public function iClickTheColumnNavigationItem($itemTitle)
+    {
+        $script = $this->clickByTitle('.column-navigation .item-text', $itemTitle);
+        $this->getSession()->executeScript($script);
+    }
+
+    /**
+     * @Then I double click the column navigation item :itemTitle
+     */
+    public function iDoubleClickTheColumnNavigationItem($itemTitle)
+    {
+        $script = $this->clickByTitle('.column-navigation .item-text', $itemTitle, 'dblclick');
+        $this->getSession()->executeScript($script);
+    }
+
+    /**
+     * Expect until all of the named events have been fired
+     * @Then I expect the following events:
+     */
+    public function iExpectTheFollowingEvents(PyStringNode $eventNames)
+    {
+        $this->waitForAuraEvents($eventNames->getStrings(), 5000);
+    }
+
+    /**
+     * @Then I expect the ":eventName" event
+     */
+    public function iExpectTheEvent($eventName)
+    {
+        $this->waitForAuraEvents(array($eventName));
+    }
+
+    /**
+     * @Then I wait a second for the ":eventName" event
+     */
+    public function iWaitASecondForTheEvent($eventName)
+    {
+        $this->waitForAuraEvents(array($eventName), 1000);
+    }
+
+    /**
+     * @Then there should be :targetNb form errors
+     */
+    public function thereShouldBeErrors($targetNb)
+    {
+        $nbErrors = $this->getSession()->evaluateScript("$('.husky-validate-error').length");
+        if ($nbErrors != $targetNb) {
+            throw new \Exception(sprintf(
+                'Was expecting "%s" form errors, but got "%s"', $targetNb, $nbErrors
+            ));
+        }
+    }
+
+    /**
+     * @Given I expect the aura component ":name" to appear
+     */
+    public function iExpectTheAuraComponentToAppear($name)
+    {
+        $selector1 = 'div[data-instance-name=\\"' . $name . '\\"]';
+        $selector2 = 'div[data-aura-instance-name=\\"' . $name . '\\"]';
+        $this->getSession()->wait(self::LONG_WAIT_TIME, sprintf('$(\'%s\').children().length > 0 || $(\'%s\').children().length > 0', $selector1, $selector2));
```

Comment:
And isn't the part in the `sprintf` redundant? I would say the second check is not needed